decidim-admin/config/brakeman.ignore

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Is it safe to ignore all of these cases? Or is it something that can't be tackled in this PR for time/resource constraints? If it's so, we can create issues to review and fix in the future.

Reply:
I have created a brakeman.ignore where it was required. In some cases i am ignoring the fields from permit! as there are too many fields that need to be whitelisted (the fields of each one of the component). 
In some some cases i got false positives when a cell is being rendered (proposal / badge). 

Let's review and merge, as it already fixes a lot of changes. It would be helpful to have an extra eye on the exceptions, maybe there is someone who may have more insights addressing those.

Reply:
> Let's review and merge, as it already fixes a lot of changes. 

I agree! It's much better than what we already have, and these security issues are going to be there either way.